src/figure_library.py

```diff
@@ -138,18 +145,66 @@ def num_cell_error_figure(img, method, pixel_file=None, gaussian_file=None, V1_f
         plt.savefig(save_path, dpi = 200)
         
     plt.show()
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(description='Create a hyperparameter sweep')
```

Comment:
Is the description correct with figure_library?

Reply:
good catch